Dockerfile

```diff
@@ -1 +1 @@
-FROM quay.io/skygeario/py-skygear:canary-onbuild
+FROM quay.io/skygeario/py-skygear:v0.23.0
```

Comment:
I got the following message if i use canary-onbuild 

Invalid message: [b'2362-419E', b'', b'{"kind":"event","name":"init","param":{"config":{"http":{"host":":3000"},"app":{"name":"my_skygear_app","api_key":"my_skygear_key","master_key":"my_master_key","access_control":"role","dev_mode":true,"cors_host":"*","slave":false,"response_timeout":60},"database":{"implementation":"pq","option":"postgresql://postgres:@db/postgres?sslmode=disable"},"asset_store":{"implementation":"fs","public":false,"fs":{"url_prefix":"http://localhost:3000/files","secret":""},"s3":{"access_key":"","secret_key":"","region":"","bucket":"","url_prefix":""},"cloud":{"host":"","token":"","public_prefix":"","private_prefix":""}},"apns":{"enable":false,"type":"cert","env":"sandbox","cert_config":{"cert":"","key":""},"token_config":{"team_id":"","key_id":"","key":""}},"gcm":{"enable":false,"api_key":""},"log":{},"zmq":{"timeout":30}}}}'], assuming socket dead